### PR TITLE
FlameBinding has invalid inheritance

### PR DESCRIPTION
**Pull request by spydon**
With the new flutter, FlameBinding had an invalid inheritance (and it has a typo in the class name).
So I was going to fix that, but it seems like it is not needed.
I don't exactly understand what it was for, so please verify.

### CODE REVIEW

File: lib/flame.dart [hunk 1]
```diff
@@ -53,24 +51,5 @@ class Flame {
 
     _bundle = bundle;
   }
-
-  /// TODO verify if this is still needed (I don't think so)
-  static void initializeWidget() {
```
Comment (erickzanardo) [inline lib/flame.dart:58]: I think this is useful still, tbh I didn't even remember that we had a init method, I usually always do this `WidgetsFlutterBinding.ensureInitialized();` manually.

Comment (spydon) [reply lib/flame.dart:58]: Alright, should I remove the TODO comment?

Comment (spydon) [reply lib/flame.dart:58]: From the comment for ensureInitialized:
```
  /// You only need to call this method if you need the binding to be
  /// initialized before calling [runApp].
```
Do we need the binding initialized before runApp?

Comment (erickzanardo) [reply lib/flame.dart:58]: yeah, the ensureInitialized is needed when you need to use some APIs to make full screen or things like that before the runApp.

I think we could remove the TODO

Comment (spydon) [reply lib/flame.dart:58]: Done

**Pull request merged**